invenio_app_ils/records/serializers/access.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Ils access list serializers."""
+
+from flask import jsonify
+
+
+def patron_to_dict(patron):
+    """Serialize a patorn to dict."""
+    return dict(
+        id=patron.id,
+        metadata=dict(
+            id=patron.id,
+            email=patron.email,
+            name=patron.name,
+        )
```

Comment:
Would it make sense to use `Patron.dumps` or `Patron.dumps_loader` here to avoid duplication?